tests/models/test_hooks.py

```diff
@@ -836,6 +837,10 @@ def call(hook, fn, *args, **kwargs):
                 attr = getattr(self, h)
                 setattr(self, h, partial(call, h, attr))
 
+        @property
+        def prepare_data_per_node(self) -> bool:
+            return True
+
```

Comment:
you don't need to override the property. since the setter is defined in the DataHooks class, set `self.prepare_data_per_node = True` in the constructor